### PR TITLE
Onboard curator cluster

### LINKED ISSUE #419: Onboarding a "light" cluster
We've recently deployed an OCP 4.7 bare metal cluster for the Curator project. I'd like to manage some aspects of the cluster through this repository (e.g., the default ingress certificate, the connection with the SSO environment, etc), but there will be other cluster admins who will be "doing their own thing".  The overlay configuration would be minimal, probably something like:

```
resources:
  - ../../../base/clusterrolebindings/cluster-admins-rb
  - ../../../base/groups/cluster-admins
  - ../../../base/ingresscontrollers/default
  - ../../../base/oauths/cluster
  - ../../../base/subscriptions/web-terminal
```

Does this seem like a reasonable plan?

### PR DESCRIPTION
**Pull request by larsks**
This adds configuration for the curator cluster [1].

Closes #419

[1]: https://console-openshift-console.apps.curator.massopen.cloud/dashboards

### CODE REVIEW

**Review by tumido — COMMENTED** [review]
/lgtm

File: cluster-scope/overlays/moc/curator/oauths/moc-sso-client-secret.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: Secret
+metadata:
```
Comment (HumairAK) [inline cluster-scope/overlays/moc/curator/oauths/moc-sso-client-secret.enc.yaml:3]: @larsks can you add the following annotations, as found [here](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/moc/infra/oauths/moc-sso-client-secret.enc.yaml#L6) 

```
    annotations:
        argocd.argoproj.io/compare-options: IgnoreExtraneous
        argocd.argoproj.io/sync-options: Prune=false
```

See [here](https://github.com/operate-first/apps/issues/278) for context.

Comment (larsks) [reply cluster-scope/overlays/moc/curator/oauths/moc-sso-client-secret.enc.yaml:3]: Done.

**Review by HumairAK — APPROVED** [review]
/lgtm

**Pull request merged**